docs/feature-flagging/randomization-sdk/index.md

```diff
@@ -0,0 +1,15 @@
+# Eppo Randomization SDK
+
+## Overview
+
+You can use Eppo's SDK to randomly assign subjects to your experiment variations. The SDK integrates with your existing event tracking system to log assignment events for later analysis in Eppo.
+
+The SDK assigns variations according to your experiment traffic allocation. You may choose to dialup the traffic allocation over a period of time. If the traffic allocation changes, the SDK never reassigns subjects who were already assigned to a variation. You may also use the SDK to test an experiment before launch. Each variation in Eppo has an allow-list override for test subjects. The SDK downloads this allow-list from Eppo's servers to give assignments.
```

Comment:
Suggest subbing:

`according to your experiment traffic allocation.` to
`according to the experiment traffic allocations you set in Eppo's user interface.`